### ISSUE TITLE
Import new sampling data: mcdan.us 2021-2022 (Year 2-3) 

### ISSUE THREAD

**Issue by nutnetadmin** [nutnetadmin/nutnet-data-tracker#402]

**Comment by nutnetadmin** [comment]
From Holly Martinson:

"I want to draw your attention to one issue we had in the 2022 season. We sample this site in early September, as this is peak biomass. The site is dominated in the late season by goldenrods (Solidago), but many plants were dead when we sampled. Goldenrod death was also seen elsewhere at the field station, so the issue was not unique to the study plots. Our best guess is that hot, dry weather the previous month may have contributed. Regardless of why the plants died, there was a great deal of dead matter when we sampled biomass, and that dead biomass was therefore a mix of this year’s dead and previous year’s dead matter. After collating the data in the fall, I went back to the field and verified that both the biomass and the cover data were reasonable, given the state of the plants."

I went with Holly's distinction, meaning litter is artificially inflated for 2022

Uploaded with 'import-mcdan-2021-2022.sql'
